public/index.php

```diff
@@ -12,6 +12,11 @@
     Debug::enable();
 }
 
+if (isset($_COOKIE['testcookie']) && strpos($_SERVER['HTTP_USER_AGENT'], 'GuzzleHttp') !== false) {
```

Comment:
This can be removed from the `index.php` as per: `https://github.com/OpenConext/Stepup-Deploy/pull/135` Your concern seems to be covered in that PR. As only outside of the prod mode this check will be honored